istantest.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
If `istantestd` works lets drop `istantest` out of the PR right before it gets accepted